nextcord/message.py

```diff
@@ -361,6 +363,14 @@ def to_dict(self) -> AttachmentPayload:
             result["description"] = self.description
         return result
 
+    @property
+    def flags(self) -> Optional[ApplicationFlags]:
+        """:class:`bool`: Whether the attachment has been edited using the remix feature on mobile.
```

Comment:
You seem to have mixed up this with the docstring. Shouldn't flags be constructed in `__init__` then accessed here to check the above?